File: 1480-running-sum-of-1d-array/1480-running-sum-of-1d-array.cs
public class Solution {
    public int[] RunningSum(int[] nums) {
        int[] sum = new int[nums.Length];
        
        sum[0] = nums[0];
        for (int i = 1; i < nums.Length; i++)
            sum[i] = sum[i - 1] + nums[i];
        return sum;
    }
}

File: 1480-running-sum-of-1d-array/1480-running-sum-of-1d-array.py
class Solution:
    def runningSum(self, nums: List[int]) -> List[int]:
        sum = nums.copy()
        
        sum[0] = nums[0]
        for i in range(1, len(nums)):
            sum[i] = sum[i - 1] + nums[i]
        return sum